curriculum/challenges/english/25-front-end-development/lecture-working-with-regular-expressions/6733c5dc74176e4c496d09e6.md

```diff
@@ -1,14 +1,18 @@
 ---
 id: 6733c5dc74176e4c496d09e6
-title: What Are Lookaheads and Lookbehind Assertions, and How Do They Work?
+title: What Are Lookahead and Lookbehind Assertions, and How Do They Work?
 challengeType: 19
-# videoId: nVAaxZ34khk
-dashedName: what-are-lookaheads-and-lookbehind-assertions-and-how-do-they-work
+videoId: U-XnBty3JOo
+dashedName: what-are-lookahead-and-lookbehind-assertions-and-how-do-they-work
```

Comment:
it looks like we are not changing dashed names

https://github.com/freeCodeCamp/freeCodeCamp/issues/58314#issuecomment-2613380651

Reply:
Awesome, thanks for confirming that. I'll go in and revert these changes now.